packages/launchpad/cypress/e2e/scaffold-project.cy.ts

```diff
@@ -44,6 +44,7 @@ function scaffoldAndOpenE2EProject (opts: {
   cy.contains('E2E Testing').click()
   cy.contains('We added the following files to your project:')
   cy.contains('Continue').click()
+  cy.contains('Choose a Browser')
```

Comment:
We weren't confirming the flow completed, which meant we were missing a failure here